src/app/series/directives/series-engagement.component.css

```diff
@@ -0,0 +1,6 @@
+/**
+ * Series engagement
+ */
+.engagement {
+  width: 100%;
```

Comment:
Because of the whole "shadow DOM" thing angular uses - yes.  And this file literally only applies to things directly in `series-engagement.component.html`.

(And we have no utility classes in Publish for things like this).